src/cli.rs

```diff
@@ -50,9 +50,12 @@ impl FromStr for Bootloader {
 #[command(args_conflicts_with_subcommands = true)]
 #[command(flatten_help = true)]
 pub struct ProveArgs {
-    #[clap(long = "with-bootloader", default_value_t = false)]
+    #[clap(long = "with-bootloader", default_value_t = true)]
     pub with_bootloader: bool,
 
+    #[clap(long = "no-bootloader", conflicts_with = "with_bootloader")]
+    pub no_bootloader: Option<bool>,
```

Comment:
I checked, I want this flag to be a boolean flag like `--with-bootloader`. I don't want the user to specify `--no-bootloader=true|false`, I just want `--no-bootloader`.